Formula/ne.rb

```diff
@@ -38,7 +53,7 @@ def install
       InsertLine
       Exit
     EOS
-    system "script", "-q", "/dev/null", bin/"ne", "--macro", macros, document
+    PTY.spawn(bin/"ne", "--macro", macros, document)
```

Comment:
May want to try to wait for process to finish with something like `Process.wait pid`

Reply:
That was the issue - I was actually able to use `sleep 1` and that was enough for it to work.  This should be ready to merge now.